db/migration/V021__create_assessments.sql

```diff
@@ -0,0 +1,137 @@
+CREATE TABLE question_types (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE assessment_submission_states (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE program_participant_roles (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE curriculum_assessments (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  description TEXT,
+  max_score INT NOT NULL,
+  max_num_submissions INT NOT NULL,
+  time_limit INT,
+  curriculum_id BIGINT NOT NULL,
+  activity_id BIGINT NOT NULL,
```

Comment:
Thinking about this more, we should be able to support assessments that aren't directly linked to a program activity, so let's edit this to make it optional.